service-automation/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,12 +1,16 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```

Comment:
Remove this file from the commit. you can keep this changes locally.

Reply:
Done